### DISCUSSION TITLE
'lc-hax' already exists and is not an empty directory [SOLVED]

### DISCUSSION THREAD

**Discussion by Juiceoctavious** [winstxnhdw/lc-hax#448] [Q&A] [answered]
I did everything right and had deleted lc-hax **foolishly**, and now when I run "**git clone --recursive --depth 50 https://github.com/winstxnhdw/lc-hax.git**"  in MINGW64:/   it then says

" **$ git clone --recursive --depth 50 https://github.com/winstxnhdw/lc-hax.git
fatal: destination path 'lc-hax' already exists and is not an empty directory.**"

Any help will be useful!

**Comment by winstxnhdw** [comment]
Yes it is. Do `explorer .`. Guarantee you it’s there

**Reply by Juiceoctavious** [reply]
Hello Winst! It was there previously, but I had unfortunately deleted it previously and was attempting to clone it again so that it would appear in that specific directory once again but haven't had any luck. Would you have any suggestions?

**Reply by winstxnhdw** [reply]
You’ll have to `cd` to the directory you want to clone it. By default it’s at C:/users/urusername/

**Reply by Juiceoctavious** [reply]
Just to clarify I typed in CMD: explorer.exe . And then I combed through my This PC > Main Drive (C:) > Users > [My name ]

**Reply by winstxnhdw** [reply]
Yeap, if git is giving you that error, that means a folder of the same name already exists at that path. You’ll have to remove it to reclone.

**Reply by Juiceoctavious** [reply]
Can you clarify how I can go about doing that for me and others who might have this issue in the future? Any help will be appreciated legend!

**Reply by winstxnhdw** [reply]
I am not sure how else better to explain this. You have an existing folder in the path you cloned in. Just delete it?

**Reply by Juiceoctavious** [reply]
Okay, coming from someone like me or others who have no experience with coding or prompt commands in CMD, just to make sure we are clear, on **CMD** in order to delete from that directory would I type in:

"**CD**  git clone --recursive --depth 50 https://github.com/winstxnhdw/lc-hax.git" ?

**Reply by winstxnhdw** [reply]
No

**Reply by winstxnhdw** [reply, marked as answer]
I don’t even know this myself. I had to search it up.

It’s

```console
rmdir /s /q lc-hax
```

**Reply by Juiceoctavious** [reply]
This came up which is so strange!
```
$ rmdir /s /q lc-hax
rmdir: failed to remove '/s': No such file or directory
rmdir: failed to remove '/q': No such file or directory
rmdir: failed to remove 'lc-hax': Directory not empty
```

**Reply by winstxnhdw** [reply]
Well, that’s because you aren’t in CMD. You are in Git bash. That would be a different command.

``` 
rm -rf lc-hax
 ```

**Reply by Juiceoctavious** [reply]
So the command for git seemed to work which allowed me to remove it from the directory giving me: 

```
$ $ git clone --recursive --depth 50 https://github.com/winstxnhdw/lc-hax.git
Cloning into 'lc-hax'...
remote: Enumerating objects: 824, done.
remote: Counting objects: 100% (824/824), done.
remote: Compressing objects: 100% (573/573), done.
remote: Total 824 (delta 425), reused 443 (delta 246), pack-reused 0
Receiving objects: 100% (824/824), 4.43 MiB | 1.69 MiB/s, done.
Resolving deltas: 100% (425/425), done.
Submodule 'submodules/SharpMonoInjectorCore' (https://github.com/winstxnhdw/SharpMonoInjectorCore.git) registered for path 'submodules/SharpMonoInjectorCore'
Cloning into 'C:/Program Files/Git/lc-hax/submodules/SharpMonoInjectorCore'...
remote: Enumerating objects: 24, done.
remote: Counting objects: 100% (24/24), done.
remote: Compressing objects: 100% (22/22), done.
remote: Total 24 (delta 0), reused 8 (delta 0), pack-reused 0
Receiving objects: 100% (24/24), 12.98 KiB | 738.00 KiB/s, done.
remote: Enumerating objects: 450, done.
remote: Counting objects: 100% (450/450), done.
remote: Compressing objects: 100% (220/220), done.
remote: Total 432 (delta 239), reused 367 (delta 204), pack-reused 0
Receiving objects: 100% (432/432), 1.64 MiB | 2.35 MiB/s, done.
Resolving deltas: 100% (239/239), completed with 10 local objects.
From https://github.com/winstxnhdw/SharpMonoInjectorCore
 * branch            7cce2b3819fa153b5b16cb3327c53cbc2a6bea38 -> FETCH_HEAD
Submodule path 'submodules/SharpMonoInjectorCore': checked out '7cce2b3819fa153b5b16cb3327c53cbc2a6bea38'
```
Succesful but for some odd reason still does not appear in  `This PC > Main Drive (C:) > Users > [My name ]
`
Any suggestions on how to redirect the file after I remove it again?

**Reply by winstxnhdw** [reply]
Just do `explorer .` and you can move it manually.

**Reply by winstxnhdw** [reply]
For whatever reason, your path is at c:/Program%20Files/Git/lc-hax

**Comment by Juiceoctavious** [comment]
No such luck, but I will call it here and mark your post about removing it as the answer. I believe something went wrong on my end when tweaking the program files in the past. I appreciate you putting in time aside to help. You have a really fun program thank you!

**Reply by winstxnhdw** [reply]
I recommend that you follow the opinionated guide next time. It’s probably a better fit for you since it defaults the install to the Desktop.

**Reply by Juiceoctavious** [reply]
Thank you! I will try that one again.
